my-pern-app/backend/src/server.ts

```diff
@@ -44,10 +44,10 @@ app.put('/api/updateMovie/:movieId', async (req, res, next)=>{
         const {newTitle, newSummary, newImdbLink, newRating} = req.body
         const sql = `
         update "movies"
-        set "title" = $2
-        set "summary" = $3
-        set "imdbLink" = $5
-        set "rating" = $5
+        set "title" = $2,
```

Comment:
Line 16:
If you properly set up your vite.config you do not need to use the cors middleware.